jsonrpc.go

```diff
@@ -991,7 +1003,7 @@ var rpcHandlers = map[string]RPCHandler{
 	"getNetworkSettings":     {Func: rpcGetNetworkSettings},
 	"setNetworkSettings":     {Func: rpcSetNetworkSettings, Params: []string{"settings"}},
 	"renewDHCPLease":         {Func: rpcRenewDHCPLease},
-	"keyboardReport":         {Func: rpcKeyboardReport, Params: []string{"modifier", "keys"}},
+	"keyboardReport":         {Func: rpcKeyboardReport, Params: []string{"modifier", "keys", "hold"}},
```

Comment:
I wonder if we need to worry about backward compatibility and use a second endpoint? e.g. leave the original line there and add another `"keyboardReport2": ` for the new api?  @ym?

Reply:
This caused issues with "Shift" key handling. I have reverted.